embulk-core/src/main/java/org/embulk/spi/ProcessTask.java

```diff
@@ -25,19 +29,24 @@ public ProcessTask(
             @JsonProperty("inputType") PluginType inputPluginType,
             @JsonProperty("outputType") PluginType outputPluginType,
             @JsonProperty("filterTypes") List<PluginType> filterPluginTypes,
+            @JsonProperty("reporterTypes") Map<Reporter.Channel, PluginType> reporterPluginTypes,
             @JsonProperty("inputTask") TaskSource inputTaskSource,
             @JsonProperty("outputTask") TaskSource outputTaskSource,
             @JsonProperty("filterTasks") List<TaskSource> filterTaskSources,
+            @JsonProperty("reportersTask") Map<Reporter.Channel, TaskSource> reportersTaskSource,
```

Comment:
Thank you for catching this. We should fix it with `reporterTasks` because we already removed `Reporters` utility.